src/app/routes/api/apiResponse.ts

```diff
@@ -58,6 +58,34 @@ export function okResponse(
 	resp.status(statusCode).end(stringify(response));
 }
 
+/**
+ * Sends success API response
+ * API means JSON
+ */
+export function okApiResponse(
```

Comment:
This function is an exact duplicate of `okResponse` apart from the fact that it doesn't serialize an object to JSON itself. The purpose of the `stringify` function used in `okResponse` however is that the produced JSON will be human-readable (due to formatting) and thus ease development when looking at plain response payloads (e.g. via _curl_).

It isn't necessary to copy a complete function and only change a slightly different return value, the frontend doesn't care about the applied formatting in development mode, as long as the response content type is correct.

-> remove `okApiResponse` and use the existing function